han/han/config.txt

# Auto Generated Config
#
# VR Model Specification file <Check SimulationInput.exe for further explanation>
#
# Property     | Example            | Explination
#--------------------------------------------------------------------------------------------
# P (path)       "P path"           Currently a max of 32 files, ask if you need more
#
# E (path)       "E path"           Path to the EEM text file
#
# T (type)       "T pdb", "T obj"   PDB: Molecules, OBJ: 3D graphics, PDBC: Custom PDB with charge.
#
# C (count)      "C n"              Number of L/R should match or exceed the count specified
#
# F (flag)       "F 0"              Bit flag: 1 = sticky grid, 2 = sticky phosphorus, 
#                                               4 = don't center, 8 = ball & stick, 16 = ?
#
# B(bind)        "B 1"              Bind the chains together within the PDB file. 0 to not bind.
#
# S (Scale)      "S 1.1"            Some PDB files have tight atom placing and the structure is sensitive
#
# L (location)   "L x y z"          Location is an offset from the center of the simulation space.
#                                      -x is left, -y is down, -z is closer to the camera 
#
# R (rotation )  "R 90 0 330"       x-axis: left to right, y-axis: vertical, z-axis: depth
#                                     x: top rotates back, y: front rotates right, z: top rotates right
#
# *Anything line that does not start with P, T, C, F, S, L or R will be ignored*
#
# There can be extra L and R that are not used. Only 'C' (count) number will be used.
    
#-------------------------------------
# B type DNA 10.5 bpt, generated by Avogadro
# Red and green
P C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\TAATAGTAGTAGCATTACTGACTG-10.5.pdb
# E C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\TAATAGTAGTAGCATTACTGACTG-10.5-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
T pdb
C 1
F 8p
S 1.0
L -148.718994140625 0  0
R 0 180 0

#-------------------------------------
# B type DNA 10.5 bpt, generated by Avogadro
# Red and green
P C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\M13mp18-start48-end48-10.5.pdb
# E C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\M13mp18-start48-end48-10.5-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
T pdb
C 1
F 8p
S 1.0
L 0 0 0
R 0 0 0

#-------------------------------------
# B type DNA 10.5 bpt, generated by Avogadro
# Red and green
P C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\TGACTGACAACATCCAATAAATCA-10.5.pdb
# E C:\Users\user\Desktop\2024.05.10_VR-Sim_Gutmann_v1.1\han\TGACTGACAACATCCAATAAATCA-10.5-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
T pdb
C 1
F 8p
S 1.0
L 148.718994140625 0  0
R 120 180 0

init_pdb/config.txt

# Auto Generated Config
#
# VR Model Specification file <Check SimulationInput.exe for further explanation>
#
# Property     | Example            | Explination
#--------------------------------------------------------------------------------------------
# P (path)       "P path"           Currently a max of 32 files, ask if you need more
#
# E (path)       "E path"           Path to the EEM text file
#
# T (type)       "T pdb", "T obj"   PDB: Molecules, OBJ: 3D graphics, PDBC: Custom PDB with charge.
#
# C (count)      "C n"              Number of L/R should match or exceed the count specified
#
# F (flag)       "F 0"              Bit flag: 1 = sticky grid, 2 = sticky phosphorus, 
#                                               4 = don't center, 8 = ball & stick, 16 = ?
#
# B(bind)        "B 1"              Bind the chains together within the PDB file. 0 to not bind.
#
# S (Scale)      "S 1.1"            Some PDB files have tight atom placing and the structure is sensitive
#
# L (location)   "L x y z"          Location is an offset from the center of the simulation space.
#                                      -x is left, -y is down, -z is closer to the camera 
#
# R (rotation )  "R 90 0 330"       x-axis: left to right, y-axis: vertical, z-axis: depth
#                                     x: top rotates back, y: front rotates right, z: top rotates right
#
# *Anything line that does not start with P, T, C, F, S, L or R will be ignored*
#
# There can be extra L and R that are not used. Only 'C' (count) number will be used.
    
#-------------------------------------
# B type DNA 10.5 bpt, generated by Avogadro
# Red and green
P D:/Han/projects/transpdb/init_pdb\new_generate_3.pdb
# E 
T pdb
C 1
F 8p
S 1.0
L 0 0  0
R 0 180 0

config.py

# # Auto Generated Config
# #
# # VR Model Specification file <Check SimulationInput.exe for further explanation>
# #
# # Property     | Example            | Explination
# #--------------------------------------------------------------------------------------------
# # P (path)       "P path"           Currently a max of 32 files, ask if you need more
# #
# # E (path)       "E path"           Path to the EEM text file
# #
# # T (type)       "T pdb", "T obj"   PDB: Molecules, OBJ: 3D graphics, PDBC: Custom PDB with charge.
# #
# # C (count)      "C n"              Number of L/R should match or exceed the count specified
# #
# # F (flag)       "F 0"              Bit flag: 1 = sticky grid, 2 = sticky phosphorus, 
# #                                               4 = don't center, 8 = ball & stick, 16 = ?
# #
# # B(bind)        "B 1"              Bind the chains together within the PDB file. 0 to not bind.
# #
# # S (Scale)      "S 1.1"            Some PDB files have tight atom placing and the structure is sensitive
# #
# # L (location)   "L x y z"          Location is an offset from the center of the simulation space.
# #                                      -x is left, -y is down, -z is closer to the camera 
# #
# # R (rotation )  "R 90 0 330"       x-axis: left to right, y-axis: vertical, z-axis: depth
# #                                     x: top rotates back, y: front rotates right, z: top rotates right
# #
# # *Anything line that does not start with P, T, C, F, S, L or R will be ignored*
# #
# # There can be extra L and R that are not used. Only 'C' (count) number will be used.
    
# #-------------------------------------
# # B type DNA 10.5 bpt, generated by Avogadro
# # Red and green
# P C:/Users/Han/Desktop\TAATAGTAGTAGCATTCCCCCCCC.pdb
# # E C:/Users/Han/Desktop\TAATAGTAGTAGCATTCCCCCCCC-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
# T pdb
# C 1
# F 8p
# S 1.0
# L -148.718994140625 0  0
# R 0 180 0
# #-------------------------------------
# # B type DNA 10.5 bpt, generated by Avogadro
# # Red and green
# P C:/Users/Han/Desktop\M13mp18-start48-end48.pdb
# # E C:/Users/Han/Desktop\M13mp18-start48-end48-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
# T pdb
# C 1
# F 8p
# S 1.0
# L 0 0 0
# R 0 0 0
# #-------------------------------------
# # B type DNA 10.5 bpt, generated by Avogadro
# # Red and green
# P C:/Users/Han/Desktop\ACTAAATAACCTACAAGGGGGGGG.pdb
# # E C:/Users/Han/Desktop\ACTAAATAACCTACAAGGGGGGGG-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
# T pdb
# C 1
# F 8p
# S 1.0
# L 148.718994140625 0  0
# R 0 0 0
# B type DNA 10.5 bpt, generated by Avogadro
# Red and green
P D:/Han/projects/transpdb/new_generate.pdb
# E C:/Users/Han/Desktop\M13mp18-start48-end48-EEM-Geidl 2015 (Cheminf_b3lyp_aim).txt
T pdb
C 1
F 8p
S 1.0
L 0 0 0
R 0 0 0